src/convertToObject.js

```diff
@@ -1,18 +1,31 @@
 'use strict';
 
 /**
- * Implement convertToObject function:
- *
- * Function takes a string with styles (see an example in [stylesString.js](./stylesString.js))
- * and returns an object where CSS properties are keys
- * and values are the values of related CSS properties (see an exampl in [test file](./convertToObject.test.js))
- *
  * @param {string} sourceString
  *
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const resultObject = {};
+
+  const sourceStringArray = sourceString
+    .replace(/\s+/g, ' ')
```

Comment:
This is nice. I forgit about this expression, but it is really a good case to use it.